djmoney/models/fields.py

```diff
@@ -354,10 +354,10 @@ def south_field_triple(self):
     def deconstruct(self):
         name, path, args, kwargs = super(MoneyField, self).deconstruct()
 
-        if self.default:
+        if self.default is not None:
             kwargs['default'] = self.default.amount
         if self.default_currency != DEFAULT_CURRENCY:
-            kwargs['default_currency'] = self.default_currency
+            kwargs['default_currency'] = '%s' % self.default_currency
```

Comment:
If currency codes are never unicode sensitive, this should be fine, but might as well be `str()`.

If currency codes *are* unicode sensitive, consider `from __future__ import unicode_literals` and `str(self.default_currency)`.

I would hope that our tests cover this.. but I'm not sure how well they perform with migration issues :/